app/views/admin/layouts/_area.html.erb

```diff
@@ -0,0 +1,12 @@
+<% x = area.coordinate.nil? ? 0 : area.coordinate.x %>
+<% y = area.coordinate.nil? ? 0 : area.coordinate.y %>
+<% type = area.class.name.downcase %>
```

Comment:
Should we put this `type` evaluation in a helper?

Reply:
In admin-ui-2, I created a method in grouping and desk called `type`, it avoids this.